src/config.rs

```diff
@@ -484,6 +491,11 @@ impl Config {
                 "rest_default_max_mempool_txs",
                 usize
             ),
+            rest_max_mempool_page_size: value_t_or_exit!(
+                m,
+                "rest_max_mempool_page_size",
+                usize
+            ),
```

Comment:
This should be on one line according to rustfmt

Reply:
I just copy-pasted the one above 😅